mix.exs

```diff
@@ -1,13 +1,23 @@
 defmodule Eredis.Mixfile do
   use Mix.Project
 
+  @source_url "https://github.com/Nordix/eredis/"
+  @version "1.3.2"
+
   def project do
     [
       app: :eredis,
-      version: "1.3.1-nordix",
+      deps: deps(),
+      description: "Non-blocking Redis client with focus on performance and robustness.",
       elixir: ">= 1.5.1",
-      start_permanent: Mix.env == :prod,
-      deps: deps()
+      package: package(),
+      source_url: @source_url,
+      start_permanent: Mix.env() == :prod,
+      version: @version,
+      docs: [
+        main: "readme",
+        extras: ["README.md", "CHANGELOG.md", "doc/eredis.md", "doc/eredis_sub.md"]
```

Comment:
Please add `api_reference: false` to remove that useless page

Reply:
Nice, now added aswell